Update Profil And Register

EkinOox opened the pull request:
Need help : Already relocate to "/" when new passwd is set. User is in waiting status, he must be redirect to /login

EkinOox committed:
Adding Update Profile

New way to interact with our own description

EkinOox committed:
Update Profil Picture & Update Register

Can now change profil picture

Can now add passwd if new user

Need help : Already relocate to "/" when new passwd is set. User is in waiting status, he must be redirect to /login

EkinOox committed:
Adding new features and bug resolved

- Redirection after inscription is now good (redirect to /login waiting for update his own status in "active" )

- Forgot passWord is now active to and all is runing well, we must also create an account gmail to put the smtp info in .env

- Adding new messages
- Resolving new routes
- Resolving issues

mathis-lambert commented on api-express/src/classes/PassForgotten.js:
hello, possible de renommer en ForgottenPassword.js ? merci :)

mathis-lambert commented on api-express/src/classes/PassForgotten.js:
```diff
@@ -0,0 +1,98 @@
+        try {
+            const user = await User.findOne({"user_email": email});
+            if (user) {
+                function generateRandomToken(length) {
```
utilise jwt pour générer les tokens

EkinOox committed:
Refactor and Update

- Refactor of the PassForgotten.js into ForgottenPassword.js

- Using JWT to create token but the "." to make the salt into the Token has been removed because with the "." pages not working well

EkinOox committed:
Adding Toast Notification

- Toasts are incredible !
